static/js/biomedical/disease/disease_chart.test.tsx

```diff
@@ -175,3 +176,26 @@ test("getCompoundDiseaseContraindication", () => {
     }
   }
 });
+
+test("getDiseaseCommonName", () => {
+  const cases: {
+    data: GraphNodes;
+    wantArray: string;
+  }[] = [
+    {
```

Comment:
we should include the following cases:
- happy case where common name is found
- there is a common name property but empty value so fallback to dcid
- there is no common name property so fallback to dcid

Reply:
Currently in the graph, we just have happy cases i.e., all diseases have a common name (which is due to the fact that they originate from the same data import which has commonName as a mandatory property). However, there are a few diseases where we don't have commonName but the Name instead. We're on the verge of updating the disease data to have name instead of commonName which would just change the property name and we would consistently have the name for all diseases on the graph. The fallback case is just if in the future, we end up adding diseases from other databases which don't have a name, but that seems very unlikely